wolfcrypt/src/sakke.c

```diff
@@ -2575,16 +2584,35 @@ static int sakke_modexp_loop(SakkeKey* key, mp_int* b, mp_int* e, mp_proj* r,
 {
     int err;
 #ifdef WC_NO_CACHE_RESISTANT
-    mp_proj* c[2] = { r, key->tmp.p2 };
+    mp_proj* c[2]
```

Comment:
Just make these the normal path too. No impact.

Reply:
:heavy_check_mark: